### PR TITLE
Fixes #3841: Accessibility: Support exposing current language to screen readers

### LINKED ISSUE #3841: Accessibility: Support exposing current language to screen readers
NVDA supports 43 languages, we should support exposing the current language to the tool.

Here is the requirement: 

> Ensure that user agents can correctly present content written in multiple languages. For all content, the default human language must be programmatically exposed when supported by the file format. Additionally, expose the language for each passage or phrase that differs from the default language. Exceptions include proper names, technical terms, and words or phrases that have been adopted into the default language.

CC: @zersiax

### PR DESCRIPTION
**Pull request by dbaeumer**
…

### CODE REVIEW

File: src/vs/workbench/electron-browser/index.html [hunk 1]
```diff
@@ -113,8 +113,20 @@
...
+				try {
+					if (config) {
+						nlsConfig = JSON.parse(config);
+					}
+				} catch (e) {
+				}
+			}
+			var uiLanguage = nlsConfig.availableLanguages['*'];
+			if (uiLanguage) {
+				window.document.getElementsByTagName('html')[0].setAttribute('lang', uiLanguage)
```
Comment (bpasero) [inline src/vs/workbench/electron-browser/index.html]: I am not sure if our language codes map directly to the expected language codes for the HTML attribute. The list of expected codes seem to be listed here: http://www.w3schools.com/tags/ref_language_codes.asp

**Comment by bpasero** [comment]
@dbaeumer as discussed, please review if our list of language codes can directly be used without changing for the HTML language attribute.

**Comment by dbaeumer** [comment]
@bpasero good catch. It looks like they use different standard codes simplified and traditional Chinese. Although it depends on how we use it at the end of the day.  Good examples for CSS are found here:
https://www.w3.org/International/questions/qa-css-lang

**Comment by dbaeumer** [comment]
Another good read is: http://www.w3.org/International/articles/language-tags/

**Comment by dbaeumer** [comment]
Reading the last document it looks like zh-CN and zh-TW are valid as well but zh-Hans and zh-Hant are preferable. I will change the code to use these for the lang attribute.

**Pull request merged**